src/resolvers/hostSystemProfile/hostSystemProfile.integration.ts

```diff
@@ -378,5 +378,175 @@ describe('host system profile', function () {
+                        major: 3,
+                        minor: 3
+                    }
+                }
+            }
+        ];
+
+        const R11 = {
+            value: {
+                name: 'RHEL',
+                major: 1,
+                minor: 1
+            },
+            count: 2
+        };
+        const R12 = {
+            value: {
+                name: 'RHEL',
+                major: 1,
+                minor: 2
+            },
+            count: 1
+        };
+        const C33 = {
+            value: {
+                name: 'CENT',
+                major: 3,
+                minor: 3
+            },
+            count: 1
+        };
+
+        test('basic query', async () => {
+            await createHosts(...hosts);
+
+            const { data, status } = await runQuery(QUERY, {}, getContext().headers);
+            expect(status).toEqual(200);
+            data.hostSystemProfile.operating_system.meta.should.eql({total: 3, count: 3});
+            data.hostSystemProfile.operating_system.data.should.eql([C33, R11, R12]);
+        });
+
+        describe('pagination', function () {
+            test('limit', async () => {
+                await createHosts(...hosts);
+
+                const { data, status } = await runQuery(QUERY, {
+                    limit: 2
+                }, getContext().headers);
+                expect(status).toEqual(200);
+
+                data.hostSystemProfile.operating_system.meta.should.eql({total: 3, count: 2});
+                data.hostSystemProfile.operating_system.data.should.eql([C33, R11]);
+            });
+
+            test('limit + offset', async () => {
+                await createHosts(...hosts);
+
+                const { data, status } = await runQuery(QUERY, {
+                    limit: 2,
+                    offset: 2
+                }, getContext().headers);
+                expect(status).toEqual(200);
+
+                data.hostSystemProfile.operating_system.meta.should.eql({total: 3, count: 1});
+                data.hostSystemProfile.operating_system.data.should.eql([R12]);
+            });
+        });
+
+        describe('ordering', function () {
+            test('ordering value DESC', async () => {
+                await createHosts(...hosts);
+
+                const { data, status } = await runQuery(QUERY, {
+                    order_by: 'value',
+                    order_how: 'DESC'
+                }, getContext().headers);
+                expect(status).toEqual(200);
+
+                data.hostSystemProfile.operating_system.meta.should.eql({total: 3, count: 3});
+                data.hostSystemProfile.operating_system.data.should.eql([R12, R11, C33]);
+            });
+        });
+
+        test('host filter', async () => {
+            await createHosts(...hosts);
+
+            const { data, status } = await runQuery(QUERY, {
+                hostFilter: {
+                    display_name: {
+                        matches: 'foo*'
```

Comment:
could you add, or change, this test to filter on a value that will filter the response to return fewer values. This test could pass if the filter is broken and not actually filtering. `{display_name: { eq: 'foo04'}}` should work